ui/src/logs/reducers/index.ts

```diff
@@ -0,0 +1,31 @@
+import {Source, Namespace, TimeRange} from 'src/types'
+import {ActionTypes, Action} from 'src/logs/actions'
+
+interface LogsState {
+  currentSource: Source | null
+  currentNamespaces: Namespace[]
+  currentNamespace: Namespace | null
+  timeRange: TimeRange
+}
+
+const defaultState = {
```

Comment:
would this merit a type?

Reply:
since it is only used as the default value for state which has a type a type here might be redundant.